src/components/TableSettingPagination.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import data from '../data/tableSetting.json';
+const TableSettingPagination = (props)=>{
+    
+      return data.map((item,index) =>
+          <div>
+              <button onClick={()=>props.setCurrentStep(index)} value={item.id}>{item.id}</button>
```

Comment:
This index starts from 0 as it maps the array's index but we have index starting from 1 in the json file.    {
      "id": 1,
      "title": "Bendable chopping board",
      "url": "./table-images/table_1.png"
    
    }

Reply:
You need to add 1 to the index as 

> <div>
>       <button onClick={() => props.setCurrentStep(index + 1)} value={item.id}>
>         {item.id}
>       </button>
>     </div>

Reply:
or pass item.id